### ISSUE TITLE
[Bug]: Accessibility Vocalisation for small cards component

### ISSUE THREAD

**Issue by afrahB** [Orange-OpenSource/ods-android#391]
### Prerequisites

- [X] I have [searched the backlog](https://github.com/Orange-OpenSource/ods-ios/issues) for duplicate or closed feature requests

### Your test device

Samsung A53

### OS Version

Android 11

### App version

1.0

### Describe the issue

Components -> Cards -> Card small

Activate clickable card
Activate Talkback
navigate on the card with Talkback
there is no vocalisation for the action of the card 

-> Each interactive element should be vocalised with the associated action

### Expected Behavior

Add vocalisation for the action when the card is clickable (Example : Click to visualise / click to see details ...)
